Makefile

```diff
@@ -54,6 +54,28 @@ apiserver: $(BINDIR)/apiserver
 $(BINDIR)/apiserver: cmd/service-catalog
 	$(DOCKER) $(GO_BUILD) -o $@ $(SC_PKG)/$^
 
+# This section contains the code generation stuff
+#################################################
+.generate_exes: $(BINDIR)/defaulter-gen $(BINDIR)/deepcopy-gen
```

Comment:
if the timestamp on the src files are newer than the  .generate_exes file then it'll rebuild.  Technically it should check the types.go/doc.go files too.